packages/graphql/schemas/cloud.graphql

```diff
@@ -668,10 +668,14 @@ type Mutation {
   Create a project in the dashboard and return its object
   """
   cloudProjectCreate(
+    campaign: String
     ciProviders: [String!]
+    cohort: String
```

Comment:
did you change these types or were they not generated and checked in and are just showing up here?

Reply:
this is from running `yarn build`. I think this is meant to update automatically when the cloud-schema updates 🤷🏻‍♀️